tests/ui_tests/test_cluster_list_page.py

```diff
@@ -209,6 +210,52 @@ def check_components_host_info(host_info: ComponentsHostRowInfo, name: str, comp
     check_host_value('components', host_info.components, components)
 
 
+# !===== Funcs =====!
+
+
+def cluster_with_service(sdk_client: ADCMClient, path) -> Cluster:
+    """
+    Prepared cluster for test: create cluster, couple services and couple components.
+    """
+    bundle = sdk_client.upload_from_fs(path)
+    cluster = bundle.cluster_create(name=f"Test cluster {random_string()}")
+    return cluster
+
+
+@allure.step("Prepare cluster and get config")
+def prepare_cluster_and_config(sdk_client: ADCMClient, path, app):
+    """Upload bundle, create cluster and get config"""
+    cluster = cluster_with_service(sdk_client, path)
+    config = ClusterConfigPage(app.driver, app.adcm.url, cluster.cluster_id).open()
+    config.wait_page_is_opened()
+    return cluster, config
+
+
+def check_default_field_values_in_configs(
+    cluster_config_page: ClusterConfigPage, config_item: WebElement, field_type: str, config
+):
+    """Check that input value in config is equal to the default one"""
+    main_config = config['config'][0]['subs'][0] if "subs" in config['config'][0] else config['config'][0]
```

Comment:
I guess it's a legacy form of extracting some "config"? :D

Reply:
no, there were two different funcs